docs/resources/uberbank_example1.ini

```diff
@@ -21,8 +21,14 @@ file-retention-level = merged_triggers
 
 ; Start and end times are needed to obey LIGO file naming conventions. Actual
 ; values are irrelevant. This can be overriden on the command line.
-start-time = 900000000
-end-time = 900010000
+; Here we use the times matching the estimated PSD so we have an immediate
+; reference on the resulting bank file name
+start-time = 1126051217
+end-time = 1137254417
+
```

Comment:
Please add a comment to explain that the options that follow are not really part of the [workflow] section, but are just stuck at the top and substituted in later. Maybe they could go in a separate section to emphasize that?